iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/helpers/SSLContextBuilder.java

```diff
@@ -136,13 +136,14 @@ public static RSAPrivateKey parsePrivateKeyString(String privateKeyPEM) throws G
 
     private static char[] generateTemporaryPassword()
```

Comment:
Can this method be moved to a shared helper instead? Looks like the same logic is being used in two files.

Reply:
Possibly, but I'd rather not make this a public method.